lib/build/nodeEngineFingerprint.ts

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright © 2019 Atomist, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { ExtractFingerprint, FP, sha256 } from "@atomist/sdm-pack-fingerprints";
+import * as _ from "lodash";
+
+export const NodeEngineType = "nodeEngine";
+
+export interface Engine {
+  name: string;
```

Comment:
The rest of the project, and rest of our projects, use four-space indent. Can you do the same?

You may just want to `npm install -g tsfmt` and run it against these files, or better yet, integrate it with your editor.